terraform/test-fixtures/transform-resource-count-basic/main.tf

```diff
@@ -1,4 +1,3 @@
 resource "aws_instance" "foo" {
     count = 3
-    value = "${aws_instance.foo.0.value}"
```

Comment:
Honestly not sure what this was testing, also the resulting expectation was wrong.